Delmar/force-app/main/default/lwc/del_caseCollaborationComponent/del_caseCollaborationComponent.js

```diff
@@ -104,26 +141,115 @@ export default class Del_commentsPage extends NavigationMixin(LightningElement)
     }
 
     /**
-    * @ author        : Rakesh Nayak
-    * @ description   : This method is used used to navigate to the user's profile on click of name of user
-    * @ params        : 'event'    - triggering event with click of name having 'id' attribute as CreatedById of that case comment
+    * @ author      : Vinaykant
+    * @ description : This Method is capturing Custom Event for closing Edit Modal Box for comment.
+    **/
+    closeEditModal() {
+        this.blnShowEditModal = false;
+    }
+
+    /**
+    * @ author      : Vinaykant
+    * @ description : This Method is capturing Custom Event for closing Delete Modal Box for comment.
+    **/
+    closeDeleteModal() {
+        this.blnShowDeleteModal = false;
+    }
+
+    /**
+    * @ author      : Vinaykant
+    * @ description : This Method is capturing Custom Event from Delete Modal Box to delete Case Comment.
     **/
-    navigateToUser(event) {
-        this[NavigationMixin.Navigate]({
-            type : 'standard__recordPage',
-            attributes : { 
-                recordId : event.target.dataset.id,
-                actionName : 'view'
+    saveDeleteChanges(event) {
+        this.blnShowDeleteModal = false;
+        // Calling an apex method named 'deleteCaseComment' to delete case comment record.
+        deleteCaseComment({ idCommentId: event.detail }).then(result => {
+            if(result.blnIsSuccess) {
+                this.showToastMessage('Success', 'success', 'Comment deleted successfully.');
+                this.blnIsLoading = true;
+                this.updateRecordView();
+            } else {
+                this.showToastMessage('Error', 'error', result.strErrorMessage);
             }
+        }).catch(error => {
+            this.handleErrors(error, 'Error Occured while deleting Comment.');
         });
+        
+    }
+
+    /**
+    * @ author      : Vinaykant
+    * @ description : This Method is capturing Custom Event from Edit Modal Box to edit Case Comment.
+    **/
+    saveEditChanges(event) {
+        this.blnShowEditModal = false;
+        let strCommentMessage = '';
+        //Checking if the edited comment is blank, if it's blank it will revert back to previous saved comment.
+        if (event.detail.commentBody === '' || event.detail.commentBody === null) {
```

Comment:
This handles empty string, null, and undefined
```suggestion
        if (event.detail.commentBody) {
```

Reply:
Okay I'll make this change.